src/ctia/status/routes.clj

```diff
@@ -0,0 +1,15 @@
+(ns ctia.status.routes
+  (:require
+   [compojure.api.sweet :refer :all]
+   [schema.core :as s]
+   [ctia.schemas.core :refer [StatusInfo]]
+   [ring.util.http-response :refer [ok]]))
+
+(defroutes status-routes
+  (context "/status" []
+           :tags ["Status"]
+           (GET "/" []
```

Comment:
The purpose of this route is to test the authentication of the user. Providing a `whoami` endpoint could be better, what do you think?

Reply:
I don't think so, the route is currently very basic but in the future we might add many more checks related to CTIA's health